examples/multi-page/intro-page.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Page, IntroductionPage } from '@department-of-veterans-affairs/va-forms-system-core';
+
+export default function FormIntroductionPage() {
+  return (
+    <>
+      <Page title="Introduction Page" nextPage="page-one">
```

Comment:
The Page component provides the navigation button functionality so that we can go to the next page of the example form. Could this be added to the IntroductionPage once we do the customizable IntroPage ticket?